tools/uninstall.sh

```diff
@@ -18,14 +18,14 @@ then
 
   if [ -f ~/.zshrc ] || [ -h ~/.zshrc ]
   then
```

Comment:
Can you change this style?
```
if [ -f ~/.zshrc ] || [ -h ~/.zshrc ]; then
```

ref: https://github.com/robbyrussell/oh-my-zsh/wiki/Coding-style-guide